_docs/hackathon-preperation-checklist.md

```diff
@@ -0,0 +1,43 @@
+# The Week Before
+- [x] testing task list
+
+- Make a team
+- Brainstorm ideas
+    - [link to project-ideas page?]
+- Update your software
+- Get used to AWS?
```

Comment:
Something along the lines of "sign up for an account with a cloud provider", and include the docs page which covers free account tiers and student discounts/accounts.
```suggestion
- Sign up for an account with a cloud provider
```